README.md

```diff
@@ -4,12 +4,23 @@ This repository contains a library of sample Photoshop plugins to help you on yo
 
 ## Samples list
 
+If you're adding a sample to the repo, feel free to add it to this list.
```

Comment:
Decided to drop this comment, I figured it might help make the list more maintainable for future plugins that are added. Let me know if that works!